### ISSUE TITLE
cdk deploy yields 'no such file' for ./lib/lambda.js

### ISSUE THREAD

**Issue by knoxilla** [troydieter/lhci-fargate#28]
Followed the docs all the way up to `cdk deploy` and ran into

```
 $ cdk deploy
Error: ENOENT: no such file or directory, open './lib/lambda.js'
    at Object.openSync (node:fs:585:3)
    at Object.readFileSync (node:fs:453:35)
    at new FargateEfsCustomResource (/Users/tknox/git/lhci-fargate/lib/efs-mount-fargate-cr.ts:28:38)
    at new LHCIStack (/Users/tknox/git/lhci-fargate/lib/lhci-stack.ts:125:22)
    at Object.<anonymous> (/Users/tknox/git/lhci-fargate/bin/lhci-fargate.ts:7:1)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Module.m._compile (/Users/tknox/git/lhci-fargate/node_modules/ts-node/src/index.ts:1455:23)
    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Object.require.extensions.<computed> [as .ts] (/Users/tknox/git/lhci-fargate/node_modules/ts-node/src/index.ts:1458:12)
    at Module.load (node:internal/modules/cjs/loader:981:32) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: './lib/lambda.js'
}
```

The EFS mount scripts under `./lib` do reference such a file:

```
 $ ack lambda.js
lib/efs-mount-fargate-cr.ts
28:      code: new lambda.InlineCode(fs.readFileSync('./lib/lambda.js', { encoding: 'utf-8' })),

lib/efs-mount-fargate-cr.js
15:            code: new lambda.InlineCode(fs.readFileSync('./lib/lambda.js', { encoding: 'utf-8' })),
```

I can see that `lambda.js` was added here on May 12

`git show a9e406a --stat`

But then deleted two commits later - same day!

`git show 7133daf --stat`

Maybe the intent was to move the js file from ./ to ./lib and it accidentally got deleted?

I've tried 

`git checkout a9e406a -- ./lambda.ja && mv ./lambda.js ./lib`

and then `cdk deploy` but I'm not sure that's doing the job.   The stack is stuck in and ECS Service healthcheck spiral - can't get the LHCI tasks to pass the healthchecks, so we keep registering/deregistering targets.  

So I suspect there is still some plumbing re the EFS mounts for the tasks that I'm missing.  Can you advise?

**Comment by knoxilla** [comment]
Also cool tool and great use of CDK.  Thanks for your help here!

**Comment by troydieter** [comment]
Hi @knoxilla - thanks for the input! I've fixed this with PR: Resolved with latest #29 

Try and let me know, working on my side deployed to the regions I support.

**Comment by troydieter** [comment]
Closing
